com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GameProjectBuilder.java

```diff
@@ -137,6 +141,7 @@ public Task<Void> create(IResource input) throws IOException, CompileExceptionEr
             builder.addOutput(input.changeExt(".arcd").disableCache());
             builder.addOutput(input.changeExt(".dmanifest").disableCache());
             builder.addOutput(input.changeExt(".public.der").disableCache());
+            builder.addOutput(input.changeExt(".graph.json").disableCache());
```

Comment:
Probably, yes!

Another thing: I was debating wether to generate it as an output from the GameProjectBuilder or if it is better to have another command line option. What do you think?

Reply:
I think it's directly related to liveupdate, so maybe it's fine to use it together.

Reply:
One thing that I'd like in the future (soon), is to have the archive step in it's own command: "java -jar bob.jar build archive bundle"

Currently, it has to be part of the `build` command, which is a bit cumbersome when developing different tasks.

We want to improve on the LU, with better archive support, with an "archive manifest", telling the archive builder which resources should go into which .zip file.
Then it will be really handy to have this information!
So I'd actually prefer to have this graph as a first class citizen in the build step.

Reply:
With that archive manifest, we won't care about all this traversal (which is cumbersome code), but can rely solely on filename patterns.

Reply:
Also, in the future, I think the graph ties into the fact that we do want to use the graph to decide what to build, in a generic way (to avoid building unrelated stuff).
But that's for another task...